openid/idtokenvalidator_test.go

```diff
@@ -283,14 +285,15 @@ func Test_getSigningKey_UsingValidToken_WhenSigningKeyGetterReturnsError(t *test
 func Test_getSigningKey_UsingValidToken_WhenSigningKeyGetterSucceeds(t *testing.T) {
 	pm, _, sm, kp, tv := createIDTokenValidator(t)
 
+	req := httptest.NewRequest(http.MethodGet, "/", nil)
```

Comment:
Same here.. since the test above already will check the request all the other tests can just continue to pass nil